### ISSUE TITLE
Problem with multi-token identifiers in TSV

### ISSUE THREAD

**Issue by reckart** [webanno/webanno#475]
Possibly several problems:
- multi-token identifiers should only have to be unique for one column (although afaik there have been several requests that it would be simpler if they just were unique per file) - yet reading a complex file where multi-token identifiers are unique-per-column yields unexpected results. Making them unique-per-file in the imported TSV seems to fix the problem
- however, when exporting that file again via TSV, then the multi-token identifiers are written unique-per-column.

Basically two options to fix this:

1. fix the exporter such that identifiers are written unique-per-column
2. fix the imported such that it correctly handles identifiers unique-per-column

See: https://groups.google.com/d/topic/webanno-user/dVr_-LFqtaA/discussion

**Comment by reckart** [comment]
No further changes to WebAnno TSV are planned.
